### PR TITLE
Resolve #111

### LINKED ISSUE #111: Weather command

### PR DESCRIPTION
**Pull request by Denio21**

### CODE REVIEW

File: languages/cs.json [hunk 1]
```diff
@@ -296,5 +296,23 @@
...
+    "command.weather.temperature":"Teplota",
+    "command.weather.condition":"Stav",
+    "command.weather.humidity":"Vlhkost",
+    "command.weather.wind":"Rychlost větru",
+    "command.weather.feelslike":"Pocitová cena",
+    "command.weather.lastchecked":"Naposledy zkontrolováno",
+    "command.weather.tommorow":"Zítřejší počasí v ",
+    "command.weather.precipitation":"Srážky",
+    "command.weather.request":"Vyžádáno od ",
+    "command.weather.error":"🛑 Zadaná lokace neexistuje"
```
Comment (cajthamlmatej) [inline languages/cs.json]: We dont probably want emojis in these translations strings, look for example at file https://github.com/cajthamlmatej/SchoolDiscordBot/blob/master/src/commands/PingCommand.js#L27. And use |.

File: languages/cs.json [hunk 2]
```diff
@@ -296,5 +296,23 @@
     "builder.annoucement.edit.value.title": "Hodnota",
     "builder.annoucement.edit.value.help": "Zadejte hodnotu typu kterou chcete upravit.",
 
-    "module.math.result":"Výsledek je: "
+    "module.math.result":"🤓 Výsledek je ",
```
Comment (cajthamlmatej) [inline languages/cs.json]: Why this line in pull request about weather?

File: languages/cs.json (hunk 1 above)
Comment (cajthamlmatej) [reply languages/cs.json]: And move these lines to all commands, dont add translations on end of page.

File: languages/en.json [hunk 3]
```diff
@@ -294,5 +294,17 @@
...
+    "command.weather.condition":"Condition",
+    "command.weather.humidity":"Humidity",
+    "command.weather.wind":"Speed of wind",
+    "command.weather.feelslike":"Perceived temperature",
+    "command.weather.lastchecked":"Last checked",
+    "command.weather.tommorow":"Tomorrow's weather in ",
+    "command.weather.precipitation":"Precipitation",
+    "command.weather.request":"Requested by ",
+    "command.weather.error":"🛑 The specified location does not exist"
+
```
Comment (cajthamlmatej) [inline languages/en.json]: Same as above in cs.json

File: src/commands/WeatherCommand.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                .addField(Translation.translate("command.weather.feelslike"), result[0].current.feelslike+" °C", true)
+                .addField(Translation.translate("command.weather.lastchecked"), result[0].current.observationtime, true)
+	            .setTimestamp()
+	            .setFooter(Translation.translate("command.weather.request") + message.author.name, message.author.avatarURL);
+
+            const weatherEmbedTom = new Discord.RichEmbed()
+	            .setColor('#BADC58')
+	            .setTitle(Translation.translate("command.weather.tommorow")+ result[0].location.name,true)
+                .addField(Translation.translate("command.weather.temperature"), "⬆️ " + result[0].forecast[2].low+" °C" +" | ⬇️ "+ result[0].forecast[2].high+" °C", true)
+                .addField(Translation.translate("command.weather.condition"),result[0].forecast[2].skytextday,true)
```
Comment (cajthamlmatej) [inline src/commands/WeatherCommand.js]: Dont we want to make variable for result[0] and result[0].forecast[2] for readability or for better performance?

File: src/commands/WeatherCommand.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        {
+            const weatherEmbedToday = new Discord.RichEmbed()
+	            .setColor('#BADC58')
+	            .setTitle(Translation.translate("command.weather.currentdate")+ result[0].location.name)
+	            .setThumbnail(result[0].current.imageUrl)
+                .addField(Translation.translate("command.weather.temperature"), result[0].current.temperature+" °C", true)
+                .addField(Translation.translate("command.weather.condition"), result[0].current.skytext, true)
+	            .addField(Translation.translate("command.weather.humidity"), result[0].current.humidity+"%", true)
+	            .addField(Translation.translate("command.weather.wind"), result[0].current.windspeed, true)
+                .addField(Translation.translate("command.weather.feelslike"), result[0].current.feelslike+" °C", true)
```
[the remark this answers is not in the document]
Comment (cajthamlmatej) [reply src/commands/WeatherCommand.js]: Spaces before .addField are not cool too.

File: src/commands/WeatherCommand.js (hunk 4 above)
Comment (Denio21) [reply src/commands/WeatherCommand.js]: no m8

**Pull request merged**